tox_extension_messages.c

```diff
@@ -147,6 +178,12 @@ bool parse_messages_packet(uint8_t const *data, size_t size,
 		it += 8;
 	}
 
+	if (messages_packet->message_type == MESSAGE_NEGOTIATE) {
```

Comment:
I know you're just following the pattern, but can we change these ifs to else ifs just to make it clearer to the reader that it can only be one of the types at a time? At first I couldn't figure out why they didn't return early (it's cause of the iterator update at the bottom).

Reply:
Commit ordering cleaned up and the whole if ladder is now a switch-case.